Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml

```diff
@@ -227,24 +227,24 @@ script:
       description: file
       isArray: false
       name: file
-      required: true
+      required: false
```

Comment:
change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255